google-cloud-clients/google-cloud-bigtable-admin/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableInstanceAdminClient.java

```diff
@@ -160,7 +159,7 @@ public void close() {
    */
   @SuppressWarnings("WeakerAccess")
   public Instance createInstance(CreateInstanceRequest request) {
-    return awaitFuture(createInstanceAsync(request));
+    return ApiExceptions.callAndTranslateApiException(createInstanceAsync(request));
```

Comment:
Hope this change won't cause some test to fail.

Reply:
It shouldn't. The implementation in callAndTranslateApiException is based on awaitFuture